apps/core/declarativewebcontainer.cpp

```diff
@@ -935,23 +934,6 @@ void DeclarativeWebContainer::initialize()
         return;
     }
 
-    bool clearTabs = m_settingManager->clearHistoryRequested();
-    int oldCount = m_model->count();
-
-    // Clear tabs immediately from the model.
-    if (clearTabs) {
-        m_model->clear();
-    }
-
-    // If data was cleared when initialized and we had tabs in previous
-    // session, reset tab model to unloaded state. DBManager emits
-    // tabsAvailable with empty list when tabs are cleared => tab model
-    // changes back to loaded and the initialize() slot gets called again.
-    if (m_settingManager->initialize() && (oldCount > 0) && clearTabs) {
-        m_model->setUnloaded();
```

Comment:
This can be dropped now from the model as it's not really in use. There's one test case but that's pretty useless of its own so let's just remove that tst_persistenttabmodel::setUnloaded() test as well.